Source/ACE/Entity/Player.cs

```diff
@@ -2345,27 +2345,27 @@ public void HandleActionUse(ObjectGuid usedItemId)
         {
             new ActionChain(this, () =>
             {
-                if (CurrentLandblock != null)
+                if (GetInventoryItem(usedItemId) != null)
                 {
-                    // Just forward our action to the appropriate user...
-                    ActionChain onUseChain = new ActionChain();
-                    CurrentLandblock.ChainOnObject(onUseChain, usedItemId, (WorldObject wo) =>
+                    WorldObject iwo = GetInventoryItem(usedItemId);
+                    ////iwo = iwo.GetObjectFromWeenieType(); // is this right? is this even needed?
+                    iwo.InternalOnUse(Session);
```

Comment:
Why is Player calling InternalOnUse?  The player is clearly not internal to the item.